src/table/utils/handle-accessibility.js

```diff
@@ -69,6 +69,13 @@ export const handleFocusoutEvent = (evt, shouldRefocus, blur) => {
   }
 };
 
-// get the object, find the patent of the confirm button in selection toolbar parent and focus that element
-export const focusConfirmButton = (element) =>
-  element.closest('.qv-object-wrapper')?.querySelector('.sel-toolbar-confirm')?.parentElement?.focus();
+export const focusSelectionToolbar = (element, keyboard, last) => {
+  const clientConfirmButton = element
```

Comment:
Don't have a better way of doing this. We could introduce some kind of inside client flag, but this works just fine

Reply:
Which confirm button is this? Would it not make sense if this is also exposed in keyboard?

Reply:
This is the confirm button in the client selection toolbar. The reason why we can focus the nebula one is that we can access the keyboard hook. keyboard don't have access to the client one